<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiddingBoxWindow</class>
 <widget class="QWidget" name="BiddingBoxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bidding box</string>
  </property>
  <widget class="QTableWidget" name="biddingTableWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>35</number>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>West</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>North</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>East</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>South</string>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="biddingButtonGroupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>583</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Bidding</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bidPassPushButton">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pass</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidDoublePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRedoublePushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>XX</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bidUndoPushButton">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidHintPushButton">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hint</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidNewPushButton">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>New</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="bidRankPushButton_1">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRankPushButton_2">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRankPushButton_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>3</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRankPushButton_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>4</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRankPushButton_5">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRankPushButton_6">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>6</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidRankPushButton_7">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="bidSuitPushButton_Clubs">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>♣</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidSuitPushButton_Diamonds">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>♦</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidSuitPushButton_Hearts">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string>♥</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidSuitPushButton_Spades">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>♠</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bidSuitPushButton_NoTrump">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>NT</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="biddingHistoryLable">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>0</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>lable</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="biddingStartGroupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>71</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string>Bidding Start</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="biddingStartRadioButton_West">
      <property name="text">
       <string>West</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="biddingStartRadioButton_North">
      <property name="text">
       <string>North</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="biddingStartRadioButton_East">
      <property name="text">
       <string>East</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="biddingStartRadioButton_South">
      <property name="text">
       <string>South</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="whoAreYouGroupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>71</width>
     <height>119</height>
    </rect>
   </property>
   <property name="title">
    <string>You are</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="whoAreYouRadioButton_West">
      <property name="text">
       <string>West</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="whoAreYouRadioButton_North">
      <property name="text">
       <string>North</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="whoAreYouRadioButton_East">
      <property name="text">
       <string>East</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="whoAreYouRadioButton_South">
      <property name="text">
       <string>South</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="handLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AJ.T5.KQT73.J764</string>
   </property>
  </widget>
  <widget class="QLabel" name="hintLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hint Result</string>
   </property>
  </widget>
  <widget class="QPushButton" name="constructHandPushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Get Hand!</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>321</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="westLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>西</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="northLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>北</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="eastLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>东</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="southLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>南</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="screenShotPushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>500</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>screenshot</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getOCRPushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Get OCR!</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="newGameConfirmCheckBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm New Game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bidRankPushButton_1</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidRankPushButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidRankPushButton_3</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidRankPushButton_4</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidRankPushButton_5</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidRankPushButton_6</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidRankPushButton_7</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidRankPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidSuitPushButton_Clubs</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidSuitPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidSuitPushButton_Diamonds</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidSuitPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidSuitPushButton_Hearts</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidSuitPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidSuitPushButton_Spades</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidSuitPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bidSuitPushButton_NoTrump</sender>
   <signal>clicked(bool)</signal>
   <receiver>BiddingBoxWindow</receiver>
   <slot>on_bidSuitPushButton_clicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>639</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_bidRankPushButton_clicked(bool)</slot>
  <slot>on_bidSuitPushButton_clicked(bool)</slot>
 </slots>
</ui>
